frontend/public/scss/v2/featured-product-cards.scss

```diff
@@ -0,0 +1,241 @@
+// sass-lint:disable mixins-before-declarations
+.featured-product-section-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  max-width: 1440px;
+  @include media-breakpoint-down(sm) {
+    width: 100%;
+    margin: 0;
+  }
+
+  .featured-product-section {
+    display: flex;
+    width: 1440px;
+    height: 520px;
+    flex-shrink: 0;
+    background-image: url('/static/images/featured-product-rectangle.svg');
+    background-size: cover;
+    flex-direction: column;
+    align-items: center;
+    @include media-breakpoint-down(sm) {
+      width: 100%;
+      height: 450px;
+      background-image: unset;
+      background-color: $home-page-grey-lite;
+    }
+
+    .featured-product-header {
+      width: 1344px;
+      margin: 0 auto;
+      padding-top: 60px;
+      flex-shrink: 0;
+      fill: $home-page-grey-lite;
+      display: flex;
+      align-content: space-between;
+      @include media-breakpoint-down(sm) {
+        width: 100%;
+        padding-top: 0;
+      }
+
+      .featured-product-header-title {
+        color: $home-page-header-blue;
+        font-size: 25px;
+        font-style: normal;
+        font-weight: 700;
+        line-height: normal;
+        @include media-breakpoint-down(sm) {
+          width: 176px;
+          font-size: 18px;
+          margin-left: 15px;
+          margin-top: 30px;
+        }
+      }
+
+      .featured-product-arrows {
+        display: inline-flex;
+        align-items: flex-start;
+        gap: 10px;
+        margin-left: auto;
+        @include media-breakpoint-down(sm) {
+          margin-right: 15px;
+          margin-top: 30px;
+        }
+
+        .arrow {
+          display: flex;
+          padding: 8px 15px;
+          align-items: flex-start;
+          gap: 10px;
+          border-radius: 3px;
+        }
+
+        .next-arrow {
+          border: 1px solid $home-page-dark-blue;
+          background: $home-page-dark-blue;
+        }
+
+        .prev-arrow {
+          border: 1px solid $home-page-border-grey;
+          background: $home-page-border-grey;
+        }
+      }
+    }
+
+    .featured-product-listing {
+      display: flex;
+      margin: 40px auto 0 auto;
+      width: 1344px;
+      height: 238px;
+      justify-content: flex-end;
+      align-items: center;
+      flex-shrink: 0;
+      padding-bottom: 17px;
+
+      @include media-breakpoint-down(sm) {
+        width: 100%;
+        margin-top: 30px;
+        align-items: unset;
+        padding-bottom: 0;
+        }
+
+      .carousel-inner {
+        overflow-y: unset;
+
+        .carousel-item {
+          gap: 24px;
+
+          .featured-product-card {
+            width: calc(25%);
+            border-radius: 5px;
+            background: white;
+            box-shadow: 0 2px 3px 0 rgba(18, 38, 49, 0.15);
+            @include media-breakpoint-down(sm) {
+              width: 282px;
```

Comment:
This might need to adjust a little bit to match with design in mobile UI

Reply:
Done so. The measurements used match the figma, I've swapped some hard measurements out for % though and I think I fixed it. Note that this is only the featured component so it will still shift left on moving the page. Once the full page is done, will get center-responsive.